src/Symfony/Component/Cache/composer.json

```diff
@@ -26,6 +26,7 @@
     "require-dev": {
         "cache/integration-tests": "dev-master",
         "doctrine/cache": "~1.6",
+        "doctrine/dbal": "~2.4",
```

Comment:
Bof... This would never end: install redis, install apcu, install... etc. No value. Read the doc instead (I'm on it).